include/roboteq_controller.hpp

```diff
@@ -13,69 +13,8 @@ namespace roboteq {
         roboteq_controller & operator= ( const roboteq_controller& ) = delete;
         ~roboteq_controller(){}
 
-        // set methods
-        void SetParameters(int16_t velocity, long accel, long decel, uint8_t channel);
```

Comment:
we actually need all these functions. Inside these functions is where we calls to `_comm->send_runtime_command` and `_comm->send_runtime_query`

Reply:
In example_main.cpp I call `motor_controller.SetVelocity(0x02, 0x0A);` and in roboteq_controller.hpp I call 
`void SetVelocity(uint16_t speed, uint8_t channel){
            _comm->download(roboteq::send_runtime_command::SET_VELOCITY, speed, channel);
        };`

Reply:
I'm still unsure about this as I don't have an output using candump